Source/WebCore/dom/TrustedTypePolicyFactory.h

```diff
@@ -58,6 +59,8 @@ class TrustedTypePolicyFactory : public ScriptWrappable, public RefCounted<Trust
     RefPtr<TrustedTypePolicy> defaultPolicy() const { return m_defaultPolicy; }
 
 private:
+    TrustedTypePolicyFactory(ScriptExecutionContext&);
+
     RefPtr<TrustedTypePolicy> m_defaultPolicy;
```

Comment:
This property is the cause of the heap-use after free, when I change it to a raw ptr it no longer crashes with the asan build.

That being said none of my attempts to fix it have worked so far.